src/nvim/message.c

```diff
@@ -595,6 +635,28 @@ bool emsgf(const char *const fmt, ...)
   return ret;
 }
 
+#define MULTILINE_BUFSIZE 8192
+
+bool emsgf_multiline(const char *const fmt, ...)
+{
+  bool ret;
+  va_list ap;
+
+  va_start(ap, fmt);
+
+  static char  errbuf[MULTILINE_BUFSIZE];
+  if (emsg_not_now()) {
+    return true;
```

Comment:
I think `va_start` must be paired with `va_end`. So this should be checked before `va_*` stuff.

Reply:
I checked the docs, you seem to be right. Also by the docs the `va_copy` in `api_set_error` seem to be unnecessary, one is allowed several `va_start/va_end` on the same arg list (noting in case we choose to reuse the pattern).